docs/user-guide/release-notes/master.rst

```diff
@@ -55,6 +55,27 @@ Known Issues
 Upgrade Instructions for 2.6.x --> master
 -----------------------------------------
 
+Upgrade the packages using:
+
+::
+
+    sudo yum update
+
+After yum completes you should migrate the database using:
+
+::
+
+    sudo -u apache pulp-manage-db
+
+After migrating the database reload the systemd process using:
+
+::
+
+    sudo systemctl daemon-reload
```

Comment:
Also this should not generally be needed, is there something about a change that is only on master currently that requires it? If so can you link to it?